src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1177,7 +1177,10 @@ private function registerValidatorMapping(ContainerBuilder $container, array $co
         foreach ($container->getParameter('kernel.bundles_metadata') as $bundle) {
             $dirname = $bundle['path'];
 
-            if ($container->fileExists($file = $dirname.'/Resources/config/validation.yml', false)) {
+            if (
+                $container->fileExists($file = $dirname.'/Resources/config/validation.yml', false) ||
+                $container->fileExists($file = $dirname.'/Resources/config/validation.yaml', false)
```

Comment:
The `.yml` file will take precedence (actually it'll probably be better to de the reverse).
Do you think it's an issue?